docs/i18n.md

```diff
@@ -37,7 +37,9 @@ Or use a build of the app with an inline editor (see above).
 
 ## For developers
 
-New keys can be added to `frontends/web/src/locales/en/app.json`.
+New keys can be added to `frontends/web/src/locales/en/app.json`. Keys for other languages **must not**
+be added manually. The same applies to the removal or modification of keys, only `en` keys should be
+removed/edited.
```

Comment:
existing `en` keys have to be updated in locize.

For bigger PR's I find it still useful to change en keys and commit to git. That way we see which keys should be manually updated in locize (after the PR is merged).